js/installversion.js

```diff
@@ -36,7 +36,7 @@ function getAssetInfo(os) {
     var assets = release.assets;
     for (var q = 0; q < assets.length; q++) {
       var asset = assets[q];
-      if (asset.name.includes(os) && release.target_commitish == 'master') {
+      if (asset.name.includes(os) && !release.draft) {
```

Comment:
Is there really no "pre-release" bit here? I don't think you'll ever see a draft here as those are only visible to authenticated team members.

Reply:
https://developer.github.com/v3/repos/releases/#response says there's a `prerelease` boolean? To be sure we could filter on both?